src/xdocs/checks.xml

```diff
@@ -192,6 +192,13 @@
             <td><a href="config_blocks.html#EmptyCatchBlock">EmptyCatchBlock</a></td>
             <td>Checks for empty catch blocks.</td>
           </tr>
+          <tr>
+            <td><a href="config_whitespace.html#ChainedMethodCallWrap">
+              ChainedMethodCallWrap</a></td>
+            <td>
+              Checks whether method calls specified by identifier pattern are wrapped.
```

Comment:
Please add "line" here, as in "line wrapped"

Reply:
Added it, done.

Reply:
Resolved.